### PR TITLE
Convert to ROS Catkin build system

### PR DESCRIPTION
**Pull request by d235j**
I'm not sure whether this is yet good enough to merge, but this effectively allows this package to use the Catkin build system. This is a prerequisite for https://github.com/strawlab/triggerbox/pull/2 .

### CODE REVIEW

**Comment by nzjrs** [comment]
You commited a .pyc file by mistake

(I'll take a look at the rest of this tomorrow)

**Comment by d235j** [comment]
Sorry, not sure how I missed that. Fixed.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+setup_args = generate_distutils_setup(
+    packages=['arduinoudev'],
+    scripts=[],
+    package_dir={'': 'src'},
+    install_requires = ['arduino-mk']
```
Comment (d235j) [inline setup.py:10]: I probably should also have python-serial here.

**Comment by astraw** [comment]
Sorry I just noticed this now. I don't want to depend on ROS for arduino-udev Closing.

**Pull request closed without merging**